### ISSUE TITLE
Any slight clip of an element disables mouse input on it (GLFW)

### ISSUE THREAD

**Issue by Jimmio92** [vurtun/nuklear#263]
Hello,

I've noticed whenever I make a button and it's the _same height_ as its parent, it clips enough to make mouse input stop registering for the item. I'm not sure what's causing this.

I'm using GLFW3 from source and an up to date Nuklear.

Is this intended behavior or is it just an overly sensitive clip? It seems that it's a bug, and it might just be in the GLFW wrapping part.

**EDIT:** Also, when I make a menu bar at the top of the GLFW window, and click the menu button to open a menu, the popup box itself is the size I requested, but its contents are clipped to the parent menubar nk_begin, so the content never shows... yet I am able to click it?
I tried

``` c
struct nk_command_buffer *cb = nk_window_get_canvas(nk);
cb->use_clipping = NK_CLIPPING_OFF;
```

but it was useless.

Thanks,
Jim

P.S. I love your lib, @vurtun

**Comment by Jimmio92** [comment]
For now, I have disabled clipping by commenting out line 7452
```c
nk_draw_list_add_clip(&ctx->draw_list, nk_rect(s->x, s->y, s->w, s->h));
```

I left the OpenGL scissor test in, as it works as expected. I'm still looking around to see if I can spot the problem, but I'm very new to Nuklear, so I'm not getting anywhere fast.

Thanks,
Jim

**Comment by vurtun** [comment]
Hey,
first of it is correct behavior or you could click a button even if it is outside the scrolled window space. At least at the moment I don't know if there is any alternative way to make it work correctly.
As for the general clipping behavior bug has been fixed in commit: d8688c623f6c436a7f35617c85c62713c592f28c. 
I am not quit sure what your exact problem with the menu is. Could you check again if it is fixed now as well.
